Scripts/test-exit.py

```diff
@@ -1,9 +1,33 @@
+## This small script takes as first argument a test indicator
+## and the rest of the arguments are a command to run, with its
+## command-line arguments.
+## The script runs the command and then if the first argument is
+## (1) -ok -- returns success, ignoring the exit code of the command
+## (2) -z  -- returns success iff the exit code is non-zero
+## (3) some integer literal -- return success iff the exit code 
+##     matches the given integer
+## (4) -skip -- does not run the command and just exits with success
+
 import sys
 import subprocess
 
-ec = int(sys.argv[1])
+arg = sys.argv[1]
+if arg == "-skip":
+    print( 'Skipping' )
+    exit(0)
+
 p = subprocess.run( sys.argv[2:] )
-if p.returncode != ec :
-    print( 'Expected exit code ', ec, 'instead of ', p.returncode )
+
+if arg == "-ok":
+    exit(0)
+if arg == "-z":
+    if p.returncode == 0 :
+        print( 'Expected non-zero exit code' )
+        exit(1)
+    else:
+        exit(0)
+if p.returncode != int(arg) :
+    print( 'Expected exit code', arg, ', actual result is', p.returncode )
```

Comment:
```suggestion
    print( 'Expected exit code', arg, 'actual result is', p.returncode )
```

Since `print` includes its own commas.

Reply:
Python print does not include commas in the output (by default), so this change should not happen